pyplanet/apps/contrib/mx/views/mxsearch.py

```diff
@@ -0,0 +1,101 @@
+"""
+Mode Settings Views.
+"""
+
+from pyplanet.views.generics import ManualListView
+
+
+class MxSearchListView(ManualListView):
+	title = 'Search Mania-Exchange'
+	icon_style = 'Icons128x128_1'
+	icon_substyle = 'Buddies'
+
+	def __init__(self, app, player, maps):
+		"""
+		:param app: App config instance.
+		:param player: Player instance.
+		:type app: pyplanet.apps.contrib.mx.app.RealMX
+		:type api: pyplanet.apps.contrib.mx.api.MXApi
+		"""
+		super().__init__()
+		self.manager = app.context.ui
+		self.player = player
+		self.app = app
+		self.maps = maps
+		self.fields = [
+			{
+				'name': 'Map',
+				'index': 'name',
+				'sorting': True,
+				'searching': True,
+				'width': 60,
+				'type': 'label'
+			},
+			{
+				'name': 'Author',
+				'index': 'author',
+				'sorting': True,
+				'searching': True,
+				'width': 40,
+				'type': 'label',
+			},
+			{
+				'name': 'Environment',
+				'index': 'envir',
+				'sorting': False,
+				'searching': False,
+				'width': 15,
+				'type': 'label',
+				'safe': True
+			},
+			{
+				'name': 'Awards',
```

Comment:
Please change the width of the environment column a bit (you can snoop off a bit of the awards column).

Reply:
done